### PR TITLE
Bug wave2 1138 explore programs mobile issues

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_explore_programs/ucgy_fgs_explore_programs.pages_default.inc [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<?php
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_explore_programs/ucgy_fgs_explore_programs.pages_default.inc]: Let's make sure we're not featurizing the Explore Programs page. The Compare Programs page is fine, that's very tied to its params and URL; the Explore Programs view is meant to be a brick, so we can properly update the brick content above it.

File: sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js [hunk 2]
```diff
@@ -30,22 +30,22 @@ function fgsExploreUpdateFilters() {
     }
 
     // Flags have been set. Now, hide the items.
-    $(degrees).find('.program-type').filter('[class*="-hidden"]').addClass("hidden"); // if there's any flag, hide the degree
-    $(degrees).find('.program-type').not('[class*="-hidden"]').removeClass("hidden"); // only if there are no flags can we show it
+    $(degrees).find('.program-type').filter('[class*="-hidden"]').addClass("minimized"); // if there's any flag, hide the degree
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: Just so I'm clear on this, "minimized" does roughly the same thing as the "hidden" class?

Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: There are width/height transitions applied to the minimized class, but ultimately, yes the content is hidden.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_explore_programs/ucgy_fgs_explore_programs.pages_default.inc (hunk 1 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_explore_programs/ucgy_fgs_explore_programs.pages_default.inc]: δεν καταλαβαίνω (Greek: I don't understand) I didn't touch anything here...

**Pull request merged**